packages/rancher-monitoring/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: beaa76a4ac26584c6cc77c27bc0d40769fb98b26
-version: 103.0.0
+version: 103.0.1
```

Comment:
Curious why are releasing a new version of `rancher-monitoring` when `pushprox` is a main chart. Is it a sub-chart/dependency of `rancher-monitoring`?

Reply:
It's a dependecy

Reply:
Okay